packages/typography/src/elements/CodeBlock.tsx

```diff
@@ -60,26 +68,52 @@ export const CodeBlock = React.forwardRef<HTMLPreElement, ICodeBlockProps>(
         <Highlight Prism={Prism} code={code ? code.trim() : ''} language={language || 'tsx'}>
           {({ className, tokens, getLineProps, getTokenProps }) => (
             <StyledCodeBlock className={className} ref={ref} isLight={isLight} {...other}>
```

Comment:
think this might be a minor bug, but when switching from default to `isLight` back to default, the FG text sticks and doesn't change back.

[i've linked a gif screencap here to demonstrate what's happening](https://p-a39a79.f0.n0.cdn.getcloudapp.com/items/Qwu5ZNB6/df2312c1-104f-4fb4-b989-c450ca07c8d3.gif)

Reply:
@steue I meant to note in the PR description, this is a CSS ordering problem specific to Storybook. Set the controls you want and then refresh the browser in order to see the correct styling.